src/routes/Calendar.tsx

```diff
@@ -0,0 +1,67 @@
+import CalendarItem from '../features/CalendarItem/CalendarItem';
+
+const Calendar = () => {
+    const data = [
```

Comment:
Just for future reference, you should type this data as a list of EventItemProps. Otherwise code is really good!